core/src/main/resources/lib/hudson/ballColorTd.jelly

```diff
@@ -24,9 +24,9 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:x="jelly:xml" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <st:documentation>
-    Display the ball in a TD.
-    <st:attribute name="it" type="hudson.model.BallColor">
-      Color of the ball or null to skip drawing.
+    Display the build status icon in a table cell.
+    <st:attribute name="it" type="hudson.model.StatusIcon">
```

Comment:
in reality what is passed is not a ballcolor in many cases but actually a status icon